lib/app/connector.rb

```diff
@@ -95,14 +99,14 @@ def should_sync?(connector)
         last_synced = connector['_source']['last_synced']
         return true if last_synced.nil? || last_synced.empty?
 
-        last_synced = Time.parse(last_synced).utc
+        last_synced = Time.parse(last_synced)
         sync_interval = connector['_source']['scheduling']['interval']
         cron_parser = cron_parser(sync_interval)
-        cron_parser && cron_parser.next(last_synced) < Time.now.utc
+        cron_parser && cron_parser.next(last_synced) < Time.now
       end
 
       def cron_parser(cronline)
-        CronParser.new(cronline, ActiveSupport::TimeZone.new('UTC'))
```

Comment:
With removal of this line, `require 'active_support'` on top is not required anymore. But we need this to make it work unless we will encounter [this issue](https://elastic.slack.com/archives/C0D8Y4ZJ9/p1656299878658009)